src/components/Fieldset.tsx

```diff
@@ -0,0 +1,87 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/components/lbh-fieldset/_fieldset.scss";
+
+/**
+ * The proptypes for {@link FieldsetLegend}.
+ */
+export interface FieldsetLegendProps {
+  id?: string;
+  className?: string;
+  isPageHeading?: boolean;
+  children?: React.ReactNode;
+}
+
+/**
+ * The type of a valid fieldset legend component. It allows any React functional
+ * components that accept {@link FieldsetLegendProps} as proptypes.
+ */
+export type FieldsetLegendType = React.FunctionComponent<FieldsetLegendProps>;
```

Comment:
Does this get used anywhere outside of this file? If not, I don't think we need the alias.